pkg/subcmd/integration_azure.go

```diff
@@ -0,0 +1,98 @@
+package subcmd
+
+import (
+	"fmt"
+	"log/slog"
+
+	"github.com/redhat-appstudio/rhtap-cli/pkg/config"
+	"github.com/redhat-appstudio/rhtap-cli/pkg/integrations"
+	"github.com/redhat-appstudio/rhtap-cli/pkg/k8s"
+
+	"github.com/spf13/cobra"
+)
+
+// IntegrationAzure is the sub-command for the "integration azure",
+// responsible for creating and updating the Azure integration secret.
+type IntegrationAzure struct {
+	cmd    *cobra.Command // cobra command
+	logger *slog.Logger   // application logger
+	cfg    *config.Config // installer configuration
+	kube   *k8s.Kube      // kubernetes client
+
+	azureIntegration *integrations.AzureIntegration // azure integration
+
+	host         string // E.g. 'dev.azure.com'
+	token        string // API token
+	org          string // Organization name
+	clientId     string // Client ID
+	clientSecret string // Client Secret
+	tenantId     string // tenant ID
+}
+
+var _ Interface = &IntegrationAzure{}
+
+const azureIntegrationLongDesc = `
+Manages the Azure integration with RHTAP, by storing the required
+credentials required by the RHTAP services to interact with Azure.
+The credentials are stored in a Kubernetes Secret in the configured namespace
```

Comment:
Not true anymore.

`The credentials are stored in a Kubernetes Secret in the default installation namespace.`